src/Components/App.jsx

```diff
@@ -15,7 +14,7 @@ function App() {
         <Route path="/articles/:article_id" element={<Article />} />
       </Routes>
     </>
-  )
+  );
```

Comment:
Please add a link in your Home element to the articles. As it stands, it is just displaying a header and users have to manually add the path in the url to navigate to the articles